### DISCUSSION TITLE
延迟消息不是只能发到延迟消息主题吗，为什么我的测试都可以啊

### DISCUSSION THREAD

**Discussion by wushimang1** [apache/rocketmq#6717] [Q&A] [answered]
4.x和5.x都试过了，单机模式下确实都可以，这是什么问题吗

**Comment by joeCarf** [comment]
我理解延迟消息只有被写到延迟主题中才会发挥作用。由于延迟消息的实现原理是延迟主题下的消息定时重新投递，所以如果发送到普通主题的话，应该与普通消息无异。

**Reply by joeCarf** [reply]
个人理解，欢迎指正～

**Reply by wushimang1** [reply]
我试过了，发送到普通主题也是一样能定时消费的，我看文档写的是3.x和4.x不强制校验，可以混用，但是我实测5.x也可以混用，不知道为什么

**Comment by fuyou001** [comment, marked as answer]
技术上，因历史原因，当前并没有做强制限制。延迟可以发普通消息、普通消息也可以发延迟消息。
但从运维角度，建议不要混用，避免给运维带来额外的成本。

**Comment by echooymxq** [comment]
Broker应该是没有进行校验，如果是使用5.x，那么在Proxy层会进行主题消息类型的校验。

**Reply by drpmma** [reply]
是的，5.x在Proxy做了限制，当然也可以在配置选择关闭
